pkgs/development/compilers/rust/binaryBuild.nix

```diff
@@ -85,7 +75,8 @@ rec {
       license = [ licenses.mit licenses.asl20 ];
     };
 
-    buildInputs = [ makeWrapper bash ] ++ stdenv.lib.optional stdenv.isDarwin Security;
+    buildInputs = [ makeWrapper bash ]
```

Comment:
@Mic92 do you remember why you added bash here?

Reply:
@LnL7 also looks suspicious to me. It would only make a difference in a cross-compiled build. We can properly drop it.